src/enclave/enclave.h

```diff
@@ -192,19 +188,11 @@ namespace ccf
 
     CreateNodeStatus create_new_node(
       StartType start_type_,
-      ccf::StartupConfig&& ccf_config_,
+      const ccf::StartupConfig& ccf_config_,
       std::vector<uint8_t>&& startup_snapshot,
-      uint8_t* node_cert,
-      size_t node_cert_size,
-      size_t* node_cert_len,
-      uint8_t* service_cert,
-      size_t service_cert_size,
-      size_t* service_cert_len)
+      std::vector<uint8_t>& node_cert,
+      std::vector<uint8_t>& service_cert)
     {
```

Comment:
This is no longer an ECALL-compatible, arms-length, re-serialised, different-trust-boundaries call. It's just a function call! So the signature becomes more sensible, albeit with a bunch of output variables.